c7n/resources/kinesis.py

```diff
@@ -278,6 +279,48 @@ def augment(self, resources):
         return universal_augment(self.manager, super().augment(resources))
 
 
+@resources.register('kinesis-analyticsv2')
+class KinesisAnalyticsAppV2(QueryResourceManager):
+
+    class resource_type(TypeInfo):
+        service = "kinesisanalyticsv2"
+        enum_spec = ('list_applications', 'ApplicationSummaries', None)
+        detail_spec = ('describe_application', 'ApplicationName',
+                       'ApplicationName', 'ApplicationDetail')
+        name = "ApplicationName"
+        arn = id = "ApplicationARN"
+        arn_type = 'application'
+        universal_taggable = object()
+        cfn_type = 'AWS::KinesisAnalyticsV2::Application'
+
```

Comment:
add `permission_prefix = "kinesisanalytics"`

Reply:
That fixed it. Thanks @kapilt